xbmc/input/ButtonTranslator.cpp

```diff
@@ -87,7 +87,7 @@ static const ActionMapping actions[] =
     { "showsubtitles"            , ACTION_SHOW_SUBTITLES },
     { "nextsubtitle"             , ACTION_NEXT_SUBTITLE },
     { "cyclesubtitle"            , ACTION_CYCLE_SUBTITLE },
-    { "codecinfo"                , ACTION_SHOW_CODEC },
+    { "playerdebug"              , ACTION_PLAYER_DEBUG },
```

Comment:
yeah yeah, all fine with this. go ahead, just wanted to explain my intention and what actually went wrong .